src/k8s/api/v1/path.go

```diff
@@ -0,0 +1,8 @@
+package apiv1
+
+import "github.com/canonical/microcluster/rest/types"
+
+const (
+	// K8sdAPIVersion is the path prefix that will be used for the k8sd endpoints for this api version.
+	K8sdAPIVersion types.EndpointPrefix = "1.0"
```

Comment:
Is it possible to grab this version from microcluster directly? Maybe from here: https://github.com/canonical/microcluster/blob/main/internal/db/update/schema.go

Reply:
This is not the microcluster version but *our* API version